src/ashley/settings.py

```diff
@@ -0,0 +1,39 @@
+"""This module contains default settings for lti_provider application."""
+
+from django.conf import settings
+
+FORUM_BASE_PERMISSIONS = [
+    "can_see_forum",
+    "can_read_forum",
+    "can_start_new_topics",
+    "can_reply_to_topics",
+    "can_edit_own_posts",
+    "can_post_without_approval",
+    "can_vote_in_polls",
+]
+
+
+FORUM_ADMIN_PERMISSIONS = FORUM_BASE_PERMISSIONS + [
+    "can_post_announcements",
+    "can_post_stickies",
+    "can_delete_own_posts",
+    "can_create_polls",
+    "can_lock_topics",
+    "can_edit_posts",
+    "can_delete_posts",
+    "can_approve_posts",
+    "can_reply_to_locked_topics",
+]
+
+ROLE_BASED_PERMISSIONS = {
+    "administrator": FORUM_ADMIN_PERMISSIONS,
+    "instructor": FORUM_ADMIN_PERMISSIONS,
+}
+
+settings.ASHLEY_DEFAULT_FORUM_BASE_PERMISSIONS = getattr(
```

Comment:
you shouldn't modify the settings object. Why not define your own variable and import it where you need it?
```suggestion
DEFAULT_FORUM_BASE_PERMISSIONS = getattr(
```

Reply:
These are settings that ashley operators can override.
I use the same [pattern](https://stackoverflow.com/a/8429282) than in `lti_provider` and this provides default values for these settings.

The constants defined here are only for readability purpose.